collection-werekata.js

```diff
@@ -1,9 +1,20 @@
 (function () {
-
     let flyfeatures = []; // once the view is loaded the array of coords is stored here, so flyto can go through it.
     let leg = 0; // just to keep track fo where we are in the journey
     let flyview;
 
+    // Get the base map from the query string.
+    const urlParams = new URLSearchParams(window.location.search);
+    const urltoload = urlParams.get("load");
+    // Get the query string for layer json request.
+    const sortoption = urlParams.get("sort");
+    let layerQueryString = "";
+    if (sortoption === "end") {
+        layerQueryString = "?sort=end";
+    } else if (sortoption === "start") {
+        layerQueryString = "?sort=start";
+    }
```

Comment:
This could be used for controlling the order of flying by append sort=start in URL when viewing werekata flight by time. or we could put this config from the JSON feed.

Reply:
Please remove this part.

Reply:
removed